quilt/test/test_command.py

```diff
@@ -250,6 +250,13 @@ def mock_git_clone(cmd):
         assert hasattr(test, 'foo')
         assert isinstance(test.foo(), pd.DataFrame)
 
+    def test_build_yaml_syntax_error(self):
+        mydir = os.path.dirname(__file__)
+        path = os.path.join(mydir, 'data')
+        buildfilepath = os.path.join(path, 'build_bad_syntax.yml')
+        with assertRaisesRegex(self, command.CommandException, 'Syntax error .*'):
```

Comment:
.* does nothing (pls remove), since assertRaisesRegex is already a substring match and not a whole-string match:

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex

Reply:
Right.